core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -85,91 +88,100 @@
   public AlluxioMasterRestServiceHandler(@Context ServletContext context) {
     // Poor man's dependency injection through the Jersey application scope.
     mMaster =
-        (AlluxioMaster) context.getAttribute(MasterUIWebServer.ALLUXIO_MASTER_SERVLET_RESOURCE_KEY);
+        (AlluxioMaster) context.getAttribute(MasterWebServer.ALLUXIO_MASTER_SERVLET_RESOURCE_KEY);
     mBlockMaster = mMaster.getBlockMaster();
   }
 
   /**
-   * @summary get the configuration map, the keys are ordered alphabetically.
+   * @summary get the Alluxio master information
    * @return the response object
    */
   @GET
-  @Path(GET_CONFIGURATION)
-  @ReturnType("java.util.SortedMap<java.lang.String, java.lang.String>")
-  public Response getConfiguration() {
-    return RestUtils.call(new RestUtils.RestCallable<Map<String, String>>() {
+  @Path(GET_INFO)
```

Comment:
the problem with having separate endpoints is that the RPC overhead dominates ... I think a reasonable middle ground would be for the info to accept filter on what information should be filled in?